hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+Fix and improve the type signature of hypothesis.extra.numpy functions.
```

Comment:
Should I add more info?

Reply:
I've pushed a longer message - including a thankyou!